composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "pear/console_commandline": "^1.2"
```

Comment:
Are you sure it's a good idea to add a dependency? I understand that composer is a good tool to manage dependencies, but some teams don't use it, and thus you are forcing them to use this manager. Current version of this project doesn't require composer or any other external dependency (except protobuf). So changing this will bring some additional and unnecessary work for such teams.